src/utils/tools/updatingIcons/isNewIcon.ts

```diff
@@ -0,0 +1,8 @@
+const isNewIcon = (iconCreationDate: string, currentVersionDate: Date) => {
+  // icon creation date's style is: DD/MM/yyyy, we need to convert it to the standard: yyyy/DD/MM
+  // to be able to make a comparison.
+  const creationDate = new Date(iconCreationDate.split('/').reverse().join('-'))
+  return creationDate > currentVersionDate
+}
+
+export default isNewIcon
```

Comment:
This function is used to convert the date of the icons which's stored in `eos-icons.json` file to the standard, this method should be removed in the future, if `eos-icons.json` used the standard `yyyy-DD-MM`